ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -675,6 +666,7 @@ open class DrawerController: UIViewController {
                 initResizingHandleView()
                 if presentationDirection == .down {
                     containerView.addArrangedSubview(newView)
+                    containerView.layoutIfNeeded()
```

Comment:
why are we updating layout only if statement?

Reply:
When the presentationDirection == .down, i could observe that, when .none is changed to any other resizing behaviour, the resizingHandleView is added to the containVeiw UIStack with a wired animation of the resizingHandleView coming from the top of the screen. layoutIfNeeded immediately does the pending layout and the animation is avoided.

Reply:
updated the containerView.layoutIfNeeded() instruction with a comment to denote its purpose.